src/explorer/explorerTreeController.ts

```diff
@@ -11,12 +12,15 @@ import { createLogger } from '../logging';
 
 const log = createLogger('explorer controller');
 
-export default class ExplorerTreeController implements vscode.TreeDataProvider<vscode.TreeItem> {
+export default class ExplorerTreeController
+implements vscode.TreeDataProvider<vscode.TreeItem> {
```

Comment:
Here also happens a conflict between `.vscode/setting.json` and `.eslintrc`.

Reply:
It mostly a comment for myself :) to keep an eye on how vscode setting and eslint rules might expect a different style of the input.